src/accelerate/utils/launch.py

```diff
@@ -121,6 +121,7 @@ def prepare_multi_gpu_env(args: argparse.Namespace) -> Dict[str, str]:
             setattr(args, "master_addr", str(main_process_ip))
             setattr(args, "master_port", str(main_process_port))
         else:
+            setattr(args, "rdzv_backend", "c10d")
```

Comment:
This is not the right solution because if it is set to "static" a user can still do multinode. This should be set during `accelerate config` and is saved in the YAML for the user and modified there.